src/main/java/com/google/javascript/cl2dts/DeclarationGenerator.java

```diff
@@ -150,6 +163,35 @@ public static void main(String[] args) {
     System.exit(0);
   }
 
+  List<String> parseDepgraphRoots() {
```

Comment:
I think ts2dart showed us that it's better to split up earlier rather than later. This here would also make testing easier, and the depgraph parsing is really completely unrelated to the AST walking except for the narrow "what's provided" interface.

Shouldn't `cp -r` or even `rsync` also just result in a single command?